netlink-packet-route/src/rtnl/tc/message.rs

```diff
@@ -26,6 +33,30 @@ impl TcMessage {
     }
 }
 
+impl From<i32> for TcMessage {
```

Comment:
I'd prefer to have a method for this, it would be more explicit. `From` and `Into` are conversion traits, and we're not really doing a conversion, we're creating an object and initializing one of it's attributes to a certain value.

```rust
impl TcMessage {
    /// Create a new `TcMessage` with the given index
    fn with_index(index: i32) -> Self {
        // ...
    }
}
```

Reply:
fixed.